src/lib/blockchains/nxt.coffee

```diff
@@ -74,9 +74,9 @@ module.exports = (dnschain) ->
                     try
                         response = JSON.parse data
                         result.value = JSON.parse response.aliasURI
-                        cb null, result
                     catch e
                         cb e
+                    cb null, result
```

Comment:
It was actually correct the way it was before. With this change it would call the callback twice (incorrectly) if there was an exception.